package.json

```diff
@@ -68,7 +68,17 @@
     "generate:readme:toc": "markdown-toc -i README.md",
     "generate:assets": "npm run build && npm run docs",
     "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "ci:blackbox": "npm run ci:blackbox:js && npm run ci:blackbox:ts && ci:blackbox:java",
+    "ci:blackbox:js": "npm run ci:generate:blackbox:js && npm run ci:test:blackbox:js",
+    "ci:generate:blackbox:js": "node ./test/CI-blackbox/js/renderer.js",
+    "ci:test:blackbox:js": "node ./test/CI-blackbox/js/output/output.js",
+    "ci:blackbox:ts": "npm run ci:generate:blackbox:ts && npm run ci:test:blackbox:ts",
+    "ci:generate:blackbox:ts": "node ./test/CI-blackbox/ts/renderer.js",
+    "ci:test:blackbox:ts": "tsc ./test/CI-blackbox/ts/output/output.ts --outfile ./test/CI-blackbox/ts/output/output.js && node ./test/CI-blackbox/ts/output/output.js",
+    "ci:blackbox:java": "npm run ci:generate:blackbox:java && npm run ci:test:blackbox:java",
+    "ci:generate:blackbox:java": "node ./test/CI-blackbox/java/renderer.js",
+    "ci:test:blackbox:java": "javac ./test/CI-blackbox/java/output/output.java && java -cp ./test/CI-blackbox/java/output/ output"
```

Comment:
Maybe having these scripts we should write single nodejs script and run it like:

```bash
npm run ci:blackbox -- js,ts,java
```

where `js,ts,java` are a languages

What do you think? Then we can add new language in this nodejs script.

Reply:
Could be an alternative 🤔 I will look into that!

Reply:
Added a small CLI to reduce code